test/bugs-reported/bug5.chpl

```diff
@@ -0,0 +1,71 @@
+use List;
```

Comment:
Agreed. A new JIRA Issue is created here: https://chapel.atlassian.net/browse/CHAPEL-198
Will be removing this as a future test.